### DISCUSSION TITLE
Gép-gép kapcsolaton leadott bevallástervezet szerkeszthetősége a webes felületen

### DISCUSSION THREAD

**Discussion by szkomaromi-contactplus** [nav-gov-hu/eVAT#88] [Kérdések és válaszok - Gépi interfész] [answered]
A gép-gép kapcsolaton keresztül leadott bevallástervezeteket lehet még az új webes felületen szerkeszteni és azon keresztül véglegesíteni vagy a két módszer párhuzamos és nem átjárható?

**Comment by renced42** [comment, marked as answer]
Kedves @szkomaromi-contactplus 

Nem átjárható.

**Reply by szkomaromi-contactplus** [reply]
Köszönöm a gyors választ!

Nincs is tervben, hogy egyszer majd átjárható legyen ez a két rendszer?

**Reply by renced42** [reply]
Kedves @szkomaromi-contactplus 

Nincs terveben, de bármi lehet :)
Azért nincs értelme ennek, mert a gépi adatokat a könyvelő szoftverek állítják elő. Ha a feltöltött adatokat módosítod akkor azt a könyvelő programban is módostani kellene. Dupla munka.

**Reply by szkomaromi-contactplus** [reply]
Mi ERP oldalról jövünk. Reméltük, hogy nem feltétlen kell minden adatot eltárolni az ERP-ben, ami "csak" a könyvelést érinti és eddig nem kezeltük. De, ha nem átjárhatóak a rendszerszerek, akkor kénytelenek leszünk alkalmazkodni.

**Reply by renced42** [reply]
Kedves @szkomaromi-contactplus 

Írjátok le a szkenáriót, hogy látható legyen az igény.

**Reply by omachtandras** [reply]
Én felvetnék egy problémakört. Jelenleg az ERPbe nem rögzítik be a pénztárgépek bizonylatait. AP számra, napra, fizetési módra, ÁFA kulcsra összegezve rögzítenek technikai bizonylatot és könyvelik az árbevételt. Az ÁNYKs bevallás esetén ez így elég. Bár a leírás alapján számunkra még nem teljesen tiszta, hogy mi lenne az elvárás pontosan (várom, hogy lássuk miként dolgozza ezeket fel majd a NAV a januári bevalláshoz), de biztosan komoly nehézséget okozna, ha ezeket az adatokat bizonylat szinten kellene berögzíteni. (NEm beszélve arról, hogy milyen mértékű tételnövekedést okozna, ami ellen sokan ágálnak. (Ezt nem érzem én problémának, de kétségtelen, hogy ügyfél oldalon van egy olyan ellenérzés, hogy ha 10 millió tétel helyett 25 millió van a főkönyvben, akkor az felesleges és "rossz".)) (Tudom, hogy van pénztárgép napló letöltési lehetőség, de elég sok ügyfél sokadig nekifutásra sem tudja ezt használni, mert ezek a naplók (valahogy) nem teljeskörűek (nem vagyok szakértő benne, nem vettem részt ezekben a projektekben, csak az eredményről tudok.). B tervként felmerült az elmúlt időben, hogy az eÁFÁból szednénk le ezeket a bizonylatokat, ez alapján könyvelnénk és akkor vissza is tudnánk adni, de itt meg az eddig elérhető infók alapján a fizetési mód az, ami "hiányzik" az eÁFA oldalon.

**Reply by renced42** [reply]
Kedves @omachtandras 
Az eÁfa rendszerben sincs tételes OPG adat csak aggregált így az adózótól sem várjuk ezt el, mert úgy is megvan nálunk illetve a könyvelő programba sem kell tételes adatokat rögzíteni.

**Reply by omachtandras** [reply]
Tekintsétek töröltnek a hozzászólást.... Egyeztetek a kollégákkal, ügyfelekkel, hogy talán e nélkül is tudunk megoldást nyújtani. Ha nem visszatérek új kérdést nyitva erről.

Köszönöm az információt. Az lehetetlen küldetésnek tűnne, hogy ezek az aggregált adatok fizetési módot is tartalmazzanak? Tényleg nagyon egyszerűen meg lehetne akkor csinálni, hogy ezeket az adatokat lehúzzuk, áfa alapját képező bizonylatokat és könyvelési tételeket generáljunk belőle. Nagy plusz pont lenne ez az ügyfelek szemében az eÁFA hozadékaként....

**Comment by Macskafarka** [comment]
Szeretném megérteni, hogyan jött elő az, hogy az OPG-ből csak aggregált adatra van szükség az eÁFA bevalláshoz? Az én olvasatomban az ÁFA tv. 184.§ (2) c) pontja egyértelműen BIZONYLAT SZINTŰ adatokat ír elő. Én értek valamit félre?  Idézem:
184. § (2) c) "Az adóalany az állami adó- és vámhatóság által közzétett módon és adatszerkezetben gépi interfész alkalmazásával továbbítja **a fizetendő adó megállapításának** és az **adólevonási jog gyakorlásának alapjául szolgáló bizonylat szintű adatokat** és az ezen adónyilvántartásból összeállított bevallástervezetet bevallásként gépi úton jóváhagyja."

**Reply by renced42** [reply]
Kedves @Macskafarka 

A jogalkotónak biztosan nem volt a szándéka, hogy azt írja elő az adózónak, hogy az OPG adatokat tételesen rögzítse az áfa bevallásába és a könyvelő programjába. Pontosan ezért az eÁfa rendszerben sem adunk vissza tételes bizonylat szintű adatokat OPG kapcsán, kizárólag AP számra aggregált napi adatokat.

De az biztos, hogy betű szerint, így lehet értelmezni ahogy írtad. Továbbítom az illetékeseknek a jelzést, hogy ezt tisztázódjon.

**Comment by renced42** [comment]
#92 
Itt folyatódik ez a kérdés. Ezt zárom. Mert az eredeti kérdés megválaszolásra került.
